contrib/gorm.io/gorm.v1/gorm_test.go

```diff
@@ -343,3 +343,53 @@ func TestContext(t *testing.T) {
 		assert.Equal(t, testCtx.Value(key(contextKey)), ctx.Value(key(contextKey)))
 	})
 }
+
+func TestError(t *testing.T) {
+	mt := mocktracer.Start()
+	defer mt.Stop()
+
+	assertErrCheck := func(t *testing.T, mt mocktracer.Tracer, errExist bool, opts ...Option) {
+		sqltrace.Register("pgx", &stdlib.Driver{})
+		sqlDb, err := sqltrace.Open("pgx", pgConnString)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		db, err := Open(postgres.New(postgres.Config{Conn: sqlDb}), &gorm.Config{})
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		db.AutoMigrate(&Product{})
+
```

Comment:
Let's cut down some of the whitespace in this function. There are too many unnecessary blank lines.